src/rpc/mining.cpp

```diff
@@ -1034,9 +1034,7 @@ static RPCHelpMan estimatesmartfee()
             "                   higher feerate and is more likely to be sufficient for the desired\n"
             "                   target, but is not as responsive to short term drops in the\n"
             "                   prevailing fee market.  Must be one of:\n"
-            "       \"UNSET\"\n"
-            "       \"ECONOMICAL\"\n"
-            "       \"CONSERVATIVE\""},
+             "\"" + FeeModes("\"\n\"") + "\""},
```

Comment:
This changes the strings to lowercase. I'm not sure that affects anything, but in any case you'd likely also want to change the CONSERVATIVE above to that.

Reply:
Thanks, done